app/content/deployment-options/custom-deployment/release-notes.md

```diff
@@ -0,0 +1,11 @@
+## Release Notes
+
+### 21.08.19-2
+* PostgreSQL Data Base upgraded to version 12.7
```

Comment:
database is more common than data base

Reply:
Perfect, change done: `Data Base` -> `DataBase` 😁